strength_percentiles.py

```diff
@@ -4,89 +4,130 @@
+# competitors in their demographic.
 
 import urllib2
 from bs4 import BeautifulSoup
 import re
 
-# Results for New York State Powerlifting meet
-
-quote_page = 'http://meets.revolutionpowerlifting.com/results/2016-meet-results/ny-states/'
-# contains list of tuples, each tuple (squat, bench, deadlift, total),
-# represents lifts made by individual of group corresponding to dictionary key
-lifts = {'all': [],
-        'amateur': [],
-        'professional': [],
+# The page from which the program collects powerlifting meet results
+QUOTE_PAGE = 'http://meets.revolutionpowerlifting.com/results/2016-meet-results/ny-states/'
+
+# Each row of the MEET_RESULTS_TABLE contains information corresponding to
+# one lifter in the competition.  Desired information is accessed
+# through the appropriate column of the table.  Column numbers start at 0.
+# Every other column contains a whitespace string.
+GENDER_COLUMN = 1 # Column containing gender information (male or female) for
+                  # all lifters
+PROFESSIONAL_STATUS_COLUMN = 3 # ...
+EQUIPMENT_COLUMN = 5 # ...
+SQUAT_COLUMN = 15 # ...
+BENCH_COLUMN = 17 # ...
+DEADLIFT_COLUMN = 19 # ...
+
+# This dictionary will contain lists of tuples.  Each tuple will contain one
+# lifter's (squat, bench, deadlift, total). The lists divide the results
+# by the lifters' category (ex: men, amateurs) but each lifter has
+# their result repeated in four lists, ("all," one of "men" and "women," one
+# of "amateur" and "professional," and one of "raw" and "equipped")
+categorized_results = {'all': [],
+        'amateurs': [],
+        'professionals': [],
         'men': [],
         'women': [],
         'raw': [],
         'equipped': []}
 
-page = urllib2.urlopen(quote_page)
-soup = BeautifulSoup(page, 'html.parser')
-
-# list of HTML table row tag objects, each corresponds to an individual's
-# performance at this meet
-meet_results_table = soup.find_all('tr')
-
-gender_column = 1
-professional_status_column = 3
-equipment_column = 5
-squat_column = 15 # column containing lifters' squat numbers, counting from 0
-bench_column = 17 # ...
-deadlift_column = 19 # ...
-
-for result in meet_results_table: # information for one lifter
-    contents = result.contents # list of HTML objects in row
-    if contents[1].get('colspan'): # row is column headers, no data, ignore
-        continue
+def scrape_webpage_and_store_meet_results(quote_page):
+    page = urllib2.urlopen(quote_page)
+    soup = BeautifulSoup(page, 'html.parser')
+    meet_results_table = soup.find_all('tr') # all rows of meet results table,
+                                             # each corresponds to an
+                                             # individual's performance at the
+                                             # meet
+    for row in meet_results_table: # information for one lifter
+        parse_and_store_result(row)
+
+def parse_and_store_result(row):
```

Comment:
Needs a docstring.
Also, I think it would be better to organize this as simply
parse_row(...) and store_result(...)
The name parse_and_store_result itself is kind of cumbersome and already suggests 2 separate operations. So why not organize your code as such.


Alternatively, ditch the store result bit. Whatever outer code calls the parse_results stuff can just manually add the parsed contents to the dictionary.